include/allegro5/allegro_opengl.h

```diff
@@ -22,6 +22,8 @@
    extern "C" {
 #endif
 
+#include <stdint.h>
```

Comment:
I've removed the include of stdint.h. The other includes include base.h so it ended up be superfluous.